neard/res/genesis_config.json

```diff
@@ -230,7 +230,7 @@
     1,
     20
   ],
-  "total_supply": "2050000000000000000000000000000000",
+  "total_supply": "3050000000000000000000000000000000",
```

Comment:
why this is increased?

Reply:
Because EVM account is not included by default.
This total supply is actually pretty weird, as accounts here are not included.